grants_tagger_light/training/cli_args/train_args.py

```diff
@@ -0,0 +1,62 @@
+from transformers import TrainingArguments
+from dataclasses import dataclass, field
+
+
+@dataclass
+class BertMeshTrainingArguments(TrainingArguments):
```

Comment:
I am not sure what's the point here compared to simply passing those arguments to TrainingArguments and having default values in your keyword args

Reply:
Better defaults.
If I were to do it with typer, I would have to define them manually as `typer.Argument`s.